addons/marketing_campaign/marketing_campaign.py

```diff
@@ -400,8 +400,6 @@ class marketing_campaign_activity(osv.osv):
   """),
         'email_template_id': fields.many2one('mail.template', "Email Template", help='The email to send when this activity is activated'),
         'report_id': fields.many2one('ir.actions.report.xml', "Report", help='The report to generate when this activity is activated', ),
-        'report_directory_id': fields.many2one('document.directory','Directory',
-                                help="This folder is used to store the generated reports"),
```

Comment:
:+1: 
Yes .agree . sorry i forgot. now I remembered that we removed report from camaign activity for report task. so not required this field any more.

Thanks.